lib/ui/index.ts

```diff
@@ -67,33 +53,32 @@ const uiLink = (linkText, url) => {
   return terminalUISupport.color
     ? `${linkText}: ${chalk.reset.cyan(encodedUrl)}`
     : `${linkText}: ${encodedUrl}`;
-};
-
-/**
- * Returns formatted account name, account type (if applicable), and ID
- *
- * @param {number} accountId
- * @param {boolean} bold
- * @returns {string}
- */
-const uiAccountDescription = (accountId, bold = true) => {
-  const account = getAccountConfig(accountId);
-  const message = `${account.name} [${
-    HUBSPOT_ACCOUNT_TYPE_STRINGS[account.accountType]
-  }] (${account.portalId})`;
-  return bold ? chalk.bold(message) : message;
-};
+}
+
+export function uiAccountDescription(
+  accountId: number,
+  bold: boolean = true
+): string {
+  const account = getAccountConfig(accountId) as CLIAccount;
```

Comment:
Should check for null here rather than casting

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1233/commits/3ec8a544996c20de5a030ed8db00991e855fe90d